java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -1826,6 +1826,27 @@ public final ColumnVector capitalize(Scalar delimiters) {
     throw new IllegalArgumentException("Both input column and delimiters scalar should be" +
         " string type. But got column: " + type + ", scalar: " + delimiters.getType());
   }
+
+  /**
+   * Concatenates all strings in the column into one new string delimited
+   * by an optional separator string.
+   *
+   * This returns a column with one string. Any null entries are ignored unless
+   * the narep parameter specifies a replacement string (not a null value).
+   *
+   * @param separator what to insert to separate each row.
+   * @param narep what to replace nulls with
+   * @return a ColumnVector with a single string in it.
+   */
+  public final ColumnVector joinStrings(Scalar separator, Scalar narep) {
```

Comment:
Curious why not have another flavor of `joinStrings` without the `narep` if the user doesn't want to replace the nulls instead of how it is right now where the user will have to provide a `null`

Reply:
I don't see a lot of reason to add in convenience methods for something that we are not likely to us a lot. This is being added for one very specific use case as a work around until cudf can improve things in JSON parsing. One the JSON parsing is improved I don't think this API is going to be used anymore.